File: src/main/buttons/LongButton.java
package main.buttons;

public abstract class LongButton extends Button implements Comparable<LongButton> {

    public static final int WIDTH = 500;
    static final int HEIGHT = 64;

    LongButton() {
        super(WIDTH, HEIGHT);
    }

    @Override
    protected void drawButton() {
        main.stroke(255);
        main.noFill();
        main.rect(this.getX(), this.getY(), WIDTH, HEIGHT);
    }

}

File: src/main/buttons/LongButton.kt
package main.buttons

import main.Main
import processing.core.PImage

abstract class LongButton internal constructor() : Button(WIDTH, HEIGHT), Comparable<LongButton> {
    override val tintedImage: PImage? = null

    override fun drawButton() {
        Main.stroke(255)
        Main.noFill()
        Main.rect(x, y, WIDTH.toFloat(), HEIGHT.toFloat())
    }

    companion object {
        const val WIDTH = 500
        const val HEIGHT = 64
    }
}